### PR TITLE
feat(craft): Update config to support automatic releases

### PR DESCRIPTION
**Pull request by iker-barriocanal**
Updates Craft config. With this new configuration, Craft should be able to release Java & Android SDKs.

Note: draft until Craft is properly finished, see https://github.com/getsentry/craft/pull/258.

### CODE REVIEW

File: .craft.yml [hunk 1]
```diff
@@ -20,4 +18,13 @@ targets:
       maven:io.sentry:sentry-android-timber:
       maven:io.sentry:sentry-android-okhttp:
       maven:io.sentry:sentry-kotlin-extensions:
-      maven:io:sentry:sentry-android-fragment:
+  - name: maven
```
Comment (bruno-garcia) [inline .craft.yml:21]:
```suggestion
  - name: maven
```

File: .craft.yml [hunk 2]
```diff
@@ -21,3 +19,13 @@ targets:
       maven:io.sentry:sentry-android-okhttp:
       maven:io.sentry:sentry-kotlin-extensions:
       maven:io:sentry:sentry-android-fragment:
+  - name: maven
+    gradleCliPath: ./gradlew
+    mavenCliPath: scripts/mvnw
```
Comment (marandaneto) [inline .craft.yml:24]: is this accurate after moving the /mvn folder?

File: .craft.yml [hunk 3]
```diff
@@ -1,7 +1,5 @@
 minVersion: 0.23.1
```
Comment (marandaneto) [inline .craft.yml:1]: this probably has to changed after released right

File: .craft.yml (hunk 2 above)
Comment (iker-barriocanal) [reply .craft.yml:24]: Yes, `.mvn/` should be in the same directory as `.craft.yml`.

File: .craft.yml (hunk 3 above)
Comment (iker-barriocanal) [reply .craft.yml:1]: Yes, but the feature hasn't been released yet on Craft. Once it's released, I'll get this branch ready (including updating the minimum Craft version).

**Pull request merged**